### PR TITLE
Harold ux improvements to leaderboard page 2

### LINKED ISSUE #2: FEATURE: UX improvements to leaderboard page
# User Story

* As a user
* the leaderboard UI is improved
* So that I can navigate the app more easily.

### Acceptance Criteria

- [x] Move the "back" button to the top of the page
- [x] Remove the columns labeled `userCommons Id`  and  `User Id`
- [x] Change the label on the name column from `Username` to `Farmer`
- [x] Right justify the numbers in the columns `Cows Owned`,  `Cow Health`, `Cows Bought`, `Cows Sold`, and `Cow Deaths`

### Implementation Todos

Hint: look at how the `Create Restaurant` button was placed in team03 for a hint on how to place the button in that position on the page.

image

### PR DESCRIPTION
**Pull request by haroldmo**
## Overview

In this PR, the UX improvements to the leaderboard page were implemented. The "back" button is moved to the top right, the columns titled "userCommons Id" and "User Id" were removed, the column "username" was changed to "Farmer", and all the numerical fields were right justified.

## Before
image

## After
image

Closes #2

### CODE REVIEW

File: frontend/src/main/components/Leaderboard/LeaderboardTable.js
Comment (lukewli) [inline frontend/src/main/components/Leaderboard/LeaderboardTable.js:1]: Probably will have to delete all commented code. The professor made the same comment in Yash's PR.

**Review by lukewli — DISMISSED** [review]
Tests passed. Looks good.

File: frontend/src/tests/components/Leaderboard/LeaderboardTable.test.js [hunk 1]
```diff
@@ -102,13 +94,25 @@ describe("LeaderboardTable tests", () => {
...
     expect(screen.getAllByText("$1,000.00")[0]).toHaveStyle("text-align: right;");
+    expect(screen.getAllByText("12")[0]).toHaveStyle("text-align: right;");
+
+    expect(screen.getAllByText("93")[0]).toHaveStyle("text-align: right;");
+
+    expect(screen.getAllByText("11")[0]).toHaveStyle("text-align: right;");
+    expect(screen.getAllByText("10")[0]).toHaveStyle("text-align: right;");
+    expect(screen.getAllByText("9")[0]).toHaveStyle("text-align: right;");
+
+    // expect(screen.getAllByText("$1,000.00")[0]).toHaveStyle("text-align: right;");
```
Comment (pconrad) [inline frontend/src/tests/components/Leaderboard/LeaderboardTable.test.js]: Please see: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

Comment (pconrad) [inline frontend/src/tests/components/Leaderboard/LeaderboardTable.test.js]:
```suggestion
```

File: frontend/src/tests/components/Leaderboard/LeaderboardTable.test.js [hunk 2]
```diff
@@ -102,13 +94,25 @@ describe("LeaderboardTable tests", () => {
...
+
+    // expect(screen.getAllByText("$1,000.00")[0]).toHaveStyle("text-align: right;");
```
Comment (pconrad) [inline frontend/src/tests/components/Leaderboard/LeaderboardTable.test.js]:
```suggestion
```

**Review by pconrad — CHANGES_REQUESTED** [review]
As @lukewli predicted, I'm asking that you remove the commented out code.

Please see also: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: frontend/src/tests/components/Leaderboard/LeaderboardTable.test.js (hunk 2 above)
Comment (haroldmo) [reply frontend/src/tests/components/Leaderboard/LeaderboardTable.test.js]: I will fix that now. Thank you for catching that!

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**